Tigger/Models/Formats/ASCII.py

```diff
@@ -139,6 +139,19 @@ def get_ang_field(name, units=ANGULAR_UNITS):
     def getval(num, scale=1):
         return None if (num is None or len(fields) <= num) else float(fields[num]) * scale
 
+    def getval_list(num, scale=1):
+        return None if (num is None or len(fields) <= num) else [float(x) * scale for x in fields[num].split(',')]
+    
+    def triangular_index(val):
+        print("IN TRIANGULAR INDEX")
+        n = 1
+        x = n
+        print("n, x", n, x)
+        while x < val:
+            n += 1
+            x += n
+        return n
+
```

Comment:
You can use [np.tril_indices](https://docs.scipy.org/doc/numpy-1.15.0/reference/generated/numpy.tril_indices.html) here